pymeasure/adapters/__init__.py

```diff
@@ -39,10 +39,14 @@
 
 try:
     from pymeasure.adapters.serial import SerialAdapter
-    from pymeasure.adapters.prologix import PrologixAdapter
 except ImportError:
     log.warning("PySerial library could not be loaded")
 
+try:
+    from pymeasure.adapters.prologix import PrologixAdapter
+except ImportError:
+    log.warning("Prologix library could not be loaded")
```

Comment:
Good catch.
Prologix does not depend on PySerial anymore, but on pyvisa. Therefore it is better to put it with VISAAdapter in the same try clause (a few lines above).